test/tests/kernels/forcing_function/forcing_function_error_check.i

```diff
@@ -17,7 +17,7 @@
 
   [./forcing_func]
     type = ParsedFunction
-    value = "alpha*alpha*pi*pi*sin(alpha*pi*x)"
+    value = '"alpha*alpha*pi*pi*sin(alpha*pi*x)"'
```

Comment:
Not the right fix. Remove `"`

Reply:
weird, not sure what I was thinking here...

Reply:
If you look at the test, this is actually suppose to check that the function parser fails when quotes are inside the function text itself.  New parser allows double quotes around strings - so they are no longer treated as literal without surrounding single quotes.